packages/metapackage/package.json

```diff
@@ -12,16 +12,22 @@
   },
   "license": "BSD-3-Clause",
   "author": "Project Jupyter",
-  "sideEffects": false,
+  "sideEffects": [
```

Comment:
Yeah I wouldn't have expected this change.

Reply:
I did this change, this is what I wanted to ask you @fcollonval. I do not remember exactly but or the test where failing or I could not push the changes because the integrity command was asking for this change.

Reply:
I tried locally adding a new package and these changes are created when running the integrity test. So I assume that without them the CI test will fail.

Reply:
Actually the good way to fix this is to update the CSS skip list in https://github.com/jupyterlab/jupyterlab/blob/f65faadc14ee18b45d5ea06de6472d1b04cab5a0/buildutils/src/ensure-repo.ts#L145-L226

Thanks @jtpio for this one

Reply:
I added the user and user-extension packages to the list and I still have the error when pushing commits.

```=bash
$ node scripts/ensure-buildutils.js && node buildutils/lib/ensure-repo.js
{
  "@jupyterlab/metapackage": [
    "Style files not included in sideEffects in /home/carlos/Documents/jp-user/jupyterlab/packages/metapackage"
  ],
```

Reply:
Did you set `sideEffects` to `false`?

Reply:
You need to revert all changes in metapackage in addition to add the CSS skip list update.

Reply:
Yes, I set sideEffects to false.

I reverted all the changes in metapackage, but when running the ensure repo script adds all the CSS even with the user and user-extension packages in the list of skip CSS.